spark/src/test/scala/org/apache/spark/sql/delta/DeltaCDCSuite.scala

```diff
@@ -250,13 +251,11 @@ abstract class DeltaCDCSuiteBase
       // modify timestamps
       // version 0
       modifyDeltaTimestamp(deltaLog, 0, 0)
-      val tsAfterV0 = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
-        .format(new Date(1))
+      val tsAfterV0 = dateFormat.format(new Date(1))
```

Comment:
In this PR, we change so that we take into account timezone, these test failed because they didn't take into account timezones, so I'm modifying them to take into account timezone.